src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -590,7 +594,8 @@ public static String composeRegularCallableMethodHeaderSampleCode(
               .setMethodName("get")
               .setReturnType(method.outputType())
               .build();
-      boolean returnVoid = isProtoEmptyType(method.outputType());
+      boolean returnVoid =
```

Comment:
I might be missing something here, but assigning a TypeNode to a boolean should be semantically invalid.

Reply:
The purpose of `returnVoid` is to check if the RPC method return void or not. LRO method is to check LRO's `responseType`, unary method is to check `method.outputType()`.
if return void, then sample code is `future.get();`, otherwise, the sample code here is `<ResponseType> response = future.get()`.

Reply:
Yes, I'm saying that assigning a `TypeNode` to a boolean should be invalid. Does this compile?

The `returnVoid` check seems fine.